app/helpers/placements/support/organisations_helper.rb

```diff
@@ -0,0 +1,17 @@
+module Placements::Support::OrganisationsHelper
+  def no_results(search = "", filters = [])
+    if search.blank? && filters.blank?
+      I18n.t("placements.support.organisations.index.no_organisations")
+    elsif search.blank?
+      I18n.t("no_results_with_filter")
+    elsif filters.blank?
+      I18n.t("no_results", for: search)
+    else
+      I18n.t("no_results_with_filter_and_search", for: search)
+    end
+  end
+
+  def filter_options
+    %w[scitt school university lead_school].sort
```

Comment:
Just in case anything changes with the provider types is it worth changing this to:
```suggestion
    (Provider.provider_types.keys << "school").sort
```
Then you can be sure you'll always show the options for all provider types.